test/hotspot/jtreg/vmTestbase/metaspace/gc/watermark_0_1/TestDescription.java

```diff
@@ -30,9 +30,6 @@
  * VM Testbase keywords: [nonconcurrent, no_cds]
  *
  * @comment Don't run test in configurations where we can't reliably count number of metaspace triggered GCs
- * @requires vm.gc != null | !vm.opt.final.ClassUnloadingWithConcurrentMark
- * @requires vm.gc != "G1" | !vm.opt.final.ClassUnloadingWithConcurrentMark
- * @requires vm.gc != "Z"
```

Comment:
Can you explain this change, please? I can't see it in the original change.

Reply:
Thanks for catching this. This sneaked in from another backport. I removed it.